bundles/binding/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/bus/KNXBinding.java

```diff
@@ -108,7 +108,7 @@ public void removeKNXTypeMapper(KNXTypeMapper typeMapper) {
     @Override
     protected void internalReceiveCommand(String itemName, Command command) {
         logger.trace("Received command (item='{}', command='{}')", itemName, command.toString());
-        if (!isEcho(itemName, command)) {
+        if (!KNXConnection.getCompatibilityOH2() && !isEcho(itemName, command)) {
```

Comment:
> Why shouldn't this work in OH2? 

The right question would be: Why doesn't it work on OH1, if autoupdate is NOT turned off? An update event should be there as well. The fix should be somewhere in the core framework, if there is a different behavior, it should NOT be necessary to do any change in the KNX binding.